src/index/IndexImpl.Text.cpp

```diff
@@ -1022,10 +1030,11 @@ void IndexImpl::getFilteredECListForWordsWidthOne(const string& words,
   for (size_t i = 0; i < filter.size(); ++i) {
     fSet.insert(filter(i, 0));
   }
-  vector<TextRecordIndex> cids;
-  vector<Id> eids;
-  vector<Score> scores;
-  getContextEntityScoreListsForWords(words, cids, eids, scores);
+  Index::WordEntityPostings wep;
+  wep = getContextEntityScoreListsForWords(words);
```

Comment:
Please initiliaze them directly (this also happens multiple times above)
`WordEntityPostings wep = getContextEntity....()`.